smartcontract/service/wasmvm/runtime.go

```diff
@@ -156,7 +161,15 @@ func Notify(proc *exec.Process, ptr uint32, len uint32) {
 		panic(err)
 	}
 
-	notify := &event.NotifyEventInfo{self.Service.ContextRef.CurrentContext().ContractAddress, string(bs)}
+	var notify *event.NotifyEventInfo
+	nj := new(NotifyJson)
+	err = json.Unmarshal(bs, nj)
+	if err != nil {
+		notify = &event.NotifyEventInfo{self.Service.ContextRef.CurrentContext().ContractAddress, string(bs)}
+	} else {
+		notify = &event.NotifyEventInfo{self.Service.ContextRef.CurrentContext().ContractAddress, nj.States}
+	}
```

Comment:
exact those code to a function like : 
```go
func ParseEventInfo(rawBytes []byte) *NotifyEventInfo;
```
and add unit test case for this function to document the codec convention.